tests/Extensions/ElementalAreasExtensionTest.php

```diff
@@ -48,6 +49,22 @@ protected function setUp(): void
             ->set('disallowed_elements', []);
     }
 
+    protected function tearDown(): void
+    {
+        // For whatever reason, tables are not being properly cleaned between tests
+        // This was noticed when asserting an empty database table in testSingleElementalAreaCreated()
+        // where there was still data in it from testRequireDefaultRecords()
+        $schema = (new DataObject)->getSchema();
```

Comment:
```suggestion
        $schema = DataObject::getSchema();
```
Static method should be called statically.

Reply:
Updated